### PR TITLE
Fixed for formatting errors and uncompliant UTF8 documents

### PR DESCRIPTION
**Pull request by jbirbal-skydom**
Some files had bad UTF-8 compliance or unicode but needed to be UTF-8 :

-  NBSP  `U+00A0`   **NO-BREAK SPACE**
-  THSP `U+202F`   **NARROW NO-BREAK SPAC**
 among other nonstandard encoding for PDFkit to process. 

I've used `prettier` to adjust the formatting on most filetypes. There might be some edge case situations that needs to be address

- [x] Updated `.gitignore`
- [x] Removed tracked files not needed
- [x] Fixed formatting on incompliance in `clone.ts`
- [x] Remove `\t` tabs and replace with  4 space `    `
- [ ] Test edge cases that `prettier` will not work

### CODE REVIEW

**Comment by BankkRoll** [comment]
Hey @jbirbal-skydom, Thanks for the PR. Will go ahead and look into this later this week and go ahead and push after some testing. Again thanks for contributing.

File: README.md [hunk 1]
```diff
@@ -79,6 +86,17 @@ npx repo2pdf
 ### Installing and Using repo2pdf by Cloning the Repository
 
 This method involves manually cloning the repo2pdf repository and setting it up on your local machine.
+0. Dependencies:
```
Comment (BankkRoll) [inline README.md:89]: 0 isnt possible. Also this isnt friendly for each system so will remove and clean up docs later today
